### ISSUE TITLE
Quick installation guide update suggestion for 2.6.2 version with Debian Wheezy

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#428]
Originally reported of Google Code with ID 372.

```
Hello,

There are some update suggestions to the following Quick Installation Guide in version 2.6.2 for Debian Wheezy : http://docs.ganeti.org/ganeti/2.6/html/install-quick.html

Best regards,

Artur.
```
Originally added on 2013-02-12 17:44:00 +0000 UTC.

Attachments: 
[Ganeti-2.6.2_Debian-Wheezy.txt](https://storage.googleapis.com/google-code-attachments/ganeti/issue-372/comment-0/Ganeti-2.6.2_Debian-Wheezy.txt)

**Comment by Ganeti-Issues-Migrator** [comment]
```
In 'Installation software' i suggest to mention not only to copy ganeti initd script but also ganeti cron, default, hooks scripts as well.
```
Originally added on 2013-02-12 17:52:17 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I'll handle this one.
```
Originally added on 2013-02-13 14:17:09 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-10 18:08:55 +0000 UTC.

Added Labels: Type-Documentation
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-13 14:40:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit db0e97f05d8fda6da98f6ac9a34f75b2f2c2827b
Author: Guido Trotter <ultrotter@google.com>
Date:   Tue Apr 30 15:53:11 2013 +0200

    INSTALL updates for wheezy, and fixes
    
    - Remove mention of python 2.4 specific modules, as that version
      is not supported anymore
    - Add bitarray and ipaddr
    - Remove a double space
    - Add mandatory C library packages too
    - Mention the option of using drbd modules directly from your kernel
    
    This also fixes Issue 372.
    
    Signed-off-by: Guido Trotter <ultrotter@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2013-05-02 08:45:39 +0000 UTC.

Mentioned Issue with GoogleCode ID 372 has been migrated to GitHub with ID #428.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
